xml/deployment_firstboot.xml

```diff
@@ -0,0 +1,870 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE chapter PUBLIC "-//Novell//DTD NovDoc XML V1.0//EN" "novdocx.dtd"
+[
+  <!ENTITY % NOVDOC.DEACTIVATE.IDREF "INCLUDE">
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<chapter id="cha.deployment_firstboot">
+ <title>Deploying Customized Preinstallations</title>
+ <para>
+  Rolling out customized preinstallations of &productname; to a large number
+  of identical machines spares you from installing each one of them
+  separately and provides a standardized installation for the end users.
+  With &yast; Firstboot, create customized preinstallation images and
+  determine the workflow for the final personalization steps that involve
+  end user interaction (as opposed to &ay;, which allows completely
+  automated installations; for more information, see
+  <xref linkend="cha.deployment.autoinst"/>).
+ </para>
+ <para>
+  Creating a custom installation, rolling it out to your hardware, and
+  personalizing the final product involves the following steps:
+ </para>
+ <procedure>
+  <step>
+   <para>
+    Prepare the master machine whose disk needs to be cloned to the client
+    machines. For more information, refer to
+    <xref linkend="sec.fb.master"/>.
+   </para>
+  </step>
+  <step>
+   <para>
+    Customize the firstboot workflow. For more information, refer to
+    <xref
+     linkend="sec.fb.customize"/>.
+   </para>
+  </step>
+  <step>
+   <para>
+    Clone the master machine's disk and roll this image out to the clients'
+    disks. For more information, refer to <xref linkend="sec.fb.clone"/>.
+   </para>
+  </step>
+  <step>
+   <para>
+    Have the end user personalize the instance of &productname;. For more
+    information, refer to <xref linkend="sec.fb.personalize"/>.
+   </para>
+  </step>
+ </procedure>
+ <sect1 id="sec.fb.master">
+  <title>Preparing the Master Machine</title>
+
+  <para>
+   To prepare a master machine for a firstboot workflow, proceed as follows:
+  </para>
+
+  <procedure>
+   <step id="st.master1">
+    <para>
+     Insert the installation media into the master machine.
+    </para>
+   </step>
+   <step id="st.master2">
+    <para>
+     Boot the machine.
+    </para>
+   </step>
+   <step id="st.master3">
+    <para>
+     Perform a normal installation including all necessary configuration
+     steps and wait for the installed machine to boot. Also install the
```

Comment:
"Also install"? Surely it must boot first, unless I misunderstand? In which case, I would suggest "Then, install..."

Reply:
fixxored.